Client/game_sa/CAutomobileSA.h

```diff
@@ -20,39 +20,7 @@
 
 #define MAX_PASSENGER_COUNT     8
 #define MAX_DOORS               6 // also in CDamageManager
-
-namespace eCarNode
-{
-    enum
-    {
-        NONE = 0,
-        CHASSIS = 1,
-        WHEEL_RF = 2,
-        WHEEL_RM = 3,
-        WHEEL_RB = 4,
-        WHEEL_LF = 5,
-        WHEEL_LM = 6,
-        WHEEL_LB = 7,
-        DOOR_RF = 8,
-        DOOR_RR = 9,
-        DOOR_LF = 10,
-        DOOR_LR = 11,
-        BUMP_FRONT = 12,
-        BUMP_REAR = 13,
-        WING_RF = 14,
-        WING_LF = 15,
-        BONNET = 16,
-        BOOT = 17,
-        WINDSCREEN = 18,
-        EXHAUST = 19,
-        MISC_A = 20,
-        MISC_B = 21,
-        MISC_C = 22,
-        MISC_D = 23,
-        MISC_E = 24,
-        NUM_NODES
-    };
-};
+#define MAX_NODES               25
```

Comment:
`MAX_NODES` sounds vague and can be applied to many things. It should be named according to the field in which this definition is used.

Reply:
Will ``MAX_COMPONENT_NODES`` be better?

Reply:
Maybe `CAR_NODES_NUM`?

Reply:
Done